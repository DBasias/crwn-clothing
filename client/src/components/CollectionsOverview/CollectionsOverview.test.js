import React from 'react';
import { shallow } from 'enzyme';

import { CollectionsOverview } from './';

it('should render CollectionOverview component', () => {
  expect(shallow(<CollectionsOverview collections={[]} />)).toMatchSnapshot();
});
