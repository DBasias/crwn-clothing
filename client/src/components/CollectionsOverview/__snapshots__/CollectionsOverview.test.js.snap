// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render CollectionOverview component 1`] = `ShallowWrapper {}`;
